packages/block-library/src/block/edit.js

```diff
@@ -333,6 +340,15 @@ export default function ReusableBlockEdit( {
 					</ToolbarGroup>
 				</BlockControls>
 			) }
+			{ overrides ? (
+				<BlockControls>
+					<ToolbarGroup>
+						<ToolbarButton onClick={ resetOverrides }>
+							{ __( 'Reset overrides' ) }
```

Comment:
A thought. I wonder if 'Reset to original' might be a slight improvement on the copy. It fits nicely with the 'Edit original' button next to it, and is slightly less technical.